extension/background/entityTypes.js

```diff
@@ -21,9 +21,19 @@ for (const id in metadata.music) {
   }
 }
 
+export const namedPages = [];
+
+for (const id in metadata.pageNames) {
+  const item = metadata.pageNames[id];
+  for (const name of item.names) {
+    namedPages.push(name);
+  }
+}
+
 export const entityTypes = convertEntities({
   serviceName: allServiceNames,
   musicServiceName: musicServiceNames,
+  name: namedPages,
   lang: languageNames(),
   smallNumber: English.numberNames,
 });
```

Comment:
convertEntities does its own thing. It might actually be OK, but then if you look at `entityTypes.name` (it would be better to call it `.pageName`) you'll see it is an instance of Alternatives.

A good approach here is to use things like `console.log(entityTypes.name)` to see what's what.

But to add a new thing to the entity you'd do something like:

```js
export function addPageName(name) {
  entityTypes.pageNames.alternatives.push(makeWordMatcher(name));
}
```

(Note makeWordMatcher isn't exported at the moment, but could be)